src/main/java/uk/sky/cirrus/locking/Lock.java

```diff
@@ -0,0 +1,85 @@
+package uk.sky.cirrus.locking;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.sky.cirrus.locking.exception.CannotAcquireLockException;
+import uk.sky.cirrus.locking.exception.CannotReleaseLockException;
+
+import java.time.Duration;
+
+/**
+ * Each instance attempts to acquire the lock.
+ */
+public class Lock {
```

Comment:
Yea sure, will move everything to a uk.sky.cqlmigrate package and make them package private.